packages/core/package.json

```diff
@@ -263,6 +263,24 @@
                     "type": "object",
                     "description": "%AWS.configuration.description.lambda.recentlyUploaded%",
                     "default": []
+                },
```

Comment:
On another note - please confirm the other parameters that you could pass to the CLI and if that is something we would like customers to define their config file for this extension.

Reply:
We are only targeting the necessary parameters for this launch.